MaterialLib/MPL/Properties/pPengRobinson.cpp

```diff
@@ -0,0 +1,174 @@
+/**
+ * \author Norbert Grunwald
+ * \date   18.09.2017
+ * \brief
+ *
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "pPengRobinson.h"
+#include <algorithm>
+#include <cmath>
+#include "../../MathLib/poly34.h"
+#include "../mpComponent.h"
+#include "../mpPhase.h"
+#include "pUniversalConstants.h"
+
+namespace MaterialPropertyLib
+{
+/// This constructor throws an error, since the property is not
+/// implemented on the medium scale.
+PengRobinson::PengRobinson(Medium* m) : _phase(0), _component(0)
```

Comment:
Since the constructor throws an error is it possible to remove it?

Reply:
All specific properties are assigned in the selectProperty function in mpProperty.cpp. This method uses a template to define a property to an arbitrary material type (medium, phase, or component). For this reason, every property needs constructors for any of these material types.